### ISSUE TITLE
Update jazzy table of contents

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-navigation-ios#3184]
The jazzy table of contents needs to be updated to reflect #3045 and other changes:

```
WARNING: No documented top-level declarations match name "AccountManager" specified in categories file
WARNING: No documented top-level declarations match name "PointAnnotation" specified in categories file
WARNING: No documented top-level declarations match name "Style" specified in categories file
WARNING: No documented top-level declarations match name "RoadObjectsStore" specified in categories file
WARNING: No documented top-level declarations match name "RoadObjectsStoreDelegate" specified in categories file
```

#2078 or something like it would help to catch issues like this sooner.

/cc @mapbox/navigation-ios

**Comment by 1ec5** [comment]
Some of these entries will be addressed by #3367.

**Comment by 1ec5** [comment]
As of v2.0.0-rc.2:

```
WARNING: No documented top-level declarations match name "AccountManager" specified in categories file
WARNING: No documented top-level declarations match name "PointAnnotation" specified in categories file
WARNING: No documented top-level declarations match name "Style" specified in categories file
WARNING: No documented top-level declarations match name "IncorrectVisualSubtype" specified in categories file
WARNING: No documented top-level declarations match name "RoadObject.Location" specified in categories file
WARNING: No documented top-level declarations match name "RoadObject.Kind" specified in categories file
WARNING: No documented top-level declarations match name "RoadObject.EdgeLocation" specified in categories file
WARNING: No documented top-level declarations match name "RoadObject.Position" specified in categories file
WARNING: No documented top-level declarations match name "RoadObject.Identifier" specified in categories file
```

I think the `RoadObject`-related warnings are because jazzy expects e.g. `Identifier` to be in a list nested under `RoadObject`, rather than this dot syntax.
